cowprotocol/explorer
# [Solvers] Settlements Table

# Summary

Closes #234 

Added Settlements table in Solvers page

<img width="967" alt="Screen Shot 2022-10-05 at 20 19 02" src="https://user-images.githubusercontent.com/11525018/194181517-ce83e935-9d91-49a6-b8ae-6c7154f0aaa2.png">


# To Test

1.  Open the page `solvers` . You can use the header menu to navigate.
2. Click on Settlements tab
    * You'll see the settlements table.

**Notes:** For ETH cost, we need some Subgraph PR to be merged first before we can get that info into the table.
